pkg/cmd/experimental/syncgroups/ad/augmented_ldapinterface_test.go

```diff
@@ -0,0 +1,115 @@
+package ad
+
+import (
+	"reflect"
+	"testing"
+
+	"gopkg.in/ldap.v2"
+
+	"github.com/openshift/origin/pkg/auth/ldaputil"
+	"github.com/openshift/origin/pkg/auth/ldaputil/testclient"
+)
+
+const (
+	groupsBaseDN     string = "ou=groups,dc=example,dc=com"
+	groupsBaseFilter string = "objectClass=groupOfNames"
+
+	testGroupDN string = "cn=testGroup," + groupsBaseDN
+)
+
+func newTestAugmentedADLDAPInterface(client ldap.Client) *AugmentedADLDAPInterface {
```

Comment:
accept `groupsBaseDN` and `groupsBaseFilter` as args, so I don't have to know your harness when I try to use this.